app/assets/javascripts/snowcrash/modules/notifications.js.coffee

```diff
@@ -0,0 +1,12 @@
+document.addEventListener 'turbolinks:load', ->
+  $dropdown = $('[data-id="js-notifications-dropdown"]')
+
+  if $dropdown.length
+    $container = $('[data-id="js-notification-container"]')
```

Comment:
probably can handle this via CSS selector (no js-)? not sure how this plays out w/ my data-behavior comment above